awx/main/dispatch/worker/callback.py

```diff
@@ -39,16 +43,51 @@ class CallbackBrokerWorker(BaseWorker):
     '''
 
     MAX_RETRIES = 2
+    last_stats = time.time()
+    total = 0
+    last_event = ''
     prof = None
 
     def __init__(self):
         self.buff = {}
+        self.pid = os.getpid()
+        self.redis = redis.Redis.from_url(settings.BROKER_URL)
+        for key in self.redis.keys('awx_callback_receiver_statistics_*'):
+            self.redis.delete(key)
```

Comment:
could race other callback workers on init and cause a traceback.

Reply:
Key miss on delete don't raise a traceback:

```
In [4]: r.delete('foo')
Out[4]: 0
```